### DISCUSSION TITLE
[META] Discussions and issue templates

### DISCUSSION THREAD

**Discussion by Thesola10** [Artemis-chan/osu-DIVA#137] [General]
I want to avoid repeated "Failed to load ruleset" issues and declutter the issues area when it comes to suggestions/discussions, so @Artemis-chan since you have admin control over this repo, what do you think about enabling Discussions and setting up Issue templates?

**Comment by Artemis-chan** [comment]
Nice idea I enabled them and added a issue template, Feel free to modify. Let's see if this works.

**Reply by Thesola10** [reply]
I don't believe "failed to add ruleset" warrants a template, and I seem to recall some "templates" can be made to link elsewhere instead of starting an issue -- perhaps the "failed to add ruleset" template could link to debug CI jobs?

**Reply by Thesola10** [reply]
Oh wait I could just do it myself

**Comment by ghost** [comment]
it shitted my PC in 3 seconds
